browser-fingerprint/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for browser-fingerprint 0.1
+// Project: https://www.npmjs.com/package/browser-fingerprint
+// Definitions by: Karol Janyst <https://github.com/LKay>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function browserFingerprint (): string;
```

Comment:
This module also seems to have a default member on it for ES consumers. Might be worth it to declare it as:

```ts
interface BrowserFingerprint {
    (): string;
    default: BrowserFingerprint
}
declare var browserFingerprint: BrowserFingerprint;
declare namespace browserFingerprint {};

export = browserFingerprint;
```